src/misc.ts

```diff
@@ -333,16 +333,30 @@ export function getUserInitials(user?: User) {
     : undefined;
 }
 
-interface AnyEvent {
+interface AnyEventWithPropagation {
```

Comment:
yup, they should have (although not every event is cancellable) - however in here we're just telling TypeScript to accept any object that has property `stopPropagation` so that it accepts any HTML Event or SyntheticEvent from React